src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -89,32 +99,56 @@ const New = () => {
     });
   };
 
-  const onSubmit = () => {
-    if (!selectedFile) {
+  const onSubmit = (values: FileUploadForm) => {
+    if (!values.file) {
```

Comment:
I don't think this will never execute because `values.files` has type `File` and its initial value is `{}`. That's a truthy value.

It's probably better to check against some/all of the [File properties](https://developer.mozilla.org/en-US/docs/Web/API/File)